### ISSUE TITLE
Question - Can we copy/backup our rclone config?

### ISSUE THREAD

**Issue by pascalandy** [ncw/rclone#946]
Hi, 

As I want to mount the configs from a Docker container, can you tell me where they are ?
Are they embed directly within the app (/go/bin/rclone)

Thank you!

**Comment by pascalandy** [comment]
Fund the answer

`/root/.rclone.conf`

confirm this: `cat .rclone.conf`

**Comment by ncw** [comment]
rclone -v tells you the default location of the config. You can also use --config.

**Comment by ncw** [comment]
rclone -h I meant!

**Comment by emilfihlman** [comment]
rclone -h does not produce config location

**Comment by alneven** [comment]
is this an issue if I don't have this CONF file?

root@NL:~# rclone -v
2017/04/13 21:39:39 NOTICE: Config file "/root/.config/rclone/rclone.conf" not found - using defaults

**Comment by OskarStark** [comment]
> rclone -h does not produce config location

ofc it does
```
      --config string                     Config file. (default "/Users/oskar.stark/.config/rclone/rclone.conf")
```

**Comment by emilfihlman** [comment]
Well, on Linux with rclone v 1.36 it does not.

**Comment by sjurtf** [comment]
My rclone v1.36 installation on linux does. 


```
sjurtf@cornsnake:~$ rclone -V
rclone v1.36
```
```
sjurtf@cornsnake:~$ rclone -h | grep "Config file"
      --config string                     Config file. (default "/home/sjurtf/.config/rclone/rclone.conf")
```

```
sjurtf@cornsnake:~$ uname -a
Linux cornsnake 3.16.0-4-amd64 #1 SMP Debian 3.16.39-1+deb8u2 (2017-03-07) x86_64 GNU/Linux
```

**Comment by seangenabe** [comment]
For people landing here from search results, the current command is `rclone config file`

**Comment by kanutope** [comment]
Continuing on the input from @seangenabe - I was having the same question as this thread's title, with another use case in mind. Just want to share my findings.
- **context** - I had configured rclone to copy/backup from Google Drive, onto Mega.nz and locally, on my MacBook (MacOS). I had protected my `rclone config` with a (strong) password.
- **challenge** - I did want to reuse the same configuration (8 shared drives) on a Windows machine without having to reconfigure.
- **solution**
I ran `rclone.exe config file` on a *fresh install (from ZIP)* on the Windows machine , which returned  

```console
Configuration file doesn't exist, but rclone will use this path:
C:\Users\myuser\AppData\Roaming\rclone\rclone.conf
```

after which I copied the `rclone.conf` from my MacOS onto the Windows location above and my `rclone.exe copy` commands worked exactly the same way as on the Mac (`rclone copy`), with **the same config password** (!).
